QCflow/run_opt_neutral.py

```diff
@@ -119,34 +87,15 @@ def staging_opt(mol_name, mol_dic, job_type='pop_opt_n', functional='B3LYP', bas
             not_complete[f'{k}'] = basis_set1
             #add to fail fictionary
         if data.metadata['success'] == True:
-            #did the calculations work
-            if os.path.isfile(f'{k}/{k}_{job_type}.log') == False:
-                #Run the calc
+            #If the calculations worked, was it done at highest basis set
+            if data.metadata['basis_set'] == basis_set1:
+                #Run the calc at higher basis set if no
                 run_opt_n(k, mol_dic, functional, basis_set2)
                 in_progress[f'{k}'] = basis_set2
 
-            if os.path.isfile(f'{k}/{k}_{job_type}.log') == True:
-                #Then check the next basis set worked or not
-                data2 = cclib.ccopen(f'{k}/{k}_{job_type}.log').parse()
-                if data2.metadata['success'] == False:
-                    #if the calc says failed then return not complete
-                    not_complete[f'{k}'] = basis_set2
-
-                if data2.metadata['success'] == True:
-                    #checking calculation worked
-                    if os.path.isfile(f'{k}/{k}_{job_type}.log') == False:
-                        #Run the calc
-                        run_opt_n(k, mol_dic, functional, basis_set3)
-                        in_progress[f'{k}'] = basis_set3
-
-                    if os.path.isfile(f'{k}/{k}_{job_type}.log') == True:
-                        #Checking if the final basis set worked or not
-                        data3 = cclib.ccopen(f'{k}/{k}_{job_type}.log').parse()
-                        if data3.metadata['success'] == True:
-                            fully_complete[f'{k}'] = basis_set3
-                            #All calculated and fine
-                        if data3.metadata['success'] == False:
-                            not_complete[f'{k}'] = basis_set3
```

Comment:
Removed the need for the third basis set and simplified the code a little bit to make it work better